GiTinder/Controllers/SettingsController.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+using GiTinder.Data;
+using GiTinder.Models;
+
+namespace GiTinder.Controllers
+{
+
+    [Route("api/[controller]")]
```

Comment:
@FilipTeply you'll define the routes over the methods specifically and there are no routes in the documentation starting with `api/..` so this is never going to be useful